scripts/build_mod_ic.py

```diff
@@ -129,6 +129,23 @@
         },
     },
 
+    # TSSOP-16 from JEDEC MO-153AC
```

Comment:
Aha thanks. I think that's `SOP65P500X120-16N` actually as it's the same max height as the TSSOP-20. And I've tweaked the chip_shape slightly to correctly match the max dimensions in MO-153.